Lib/textwrap.py

```diff
@@ -199,6 +199,23 @@ def _fix_sentence_endings(self, chunks):
             else:
                 i += 1
 
+    def _find_width_index(self, text, width):
+        """_find_width_index(text : string, width: int)
+
+        Find at which index the text has the required width.
```

Comment:
This should probably mention `text_len()`, otherwise it might be confusing.

Reply:
Replaced with:
```
Find at which index the text has the required length, since when using a 
different text_len, this index will not be equal to the required width.
```
I've also replaced "width" by "length" since that is the term generally used (you also call it text_len).